src/main/resources/graphql/service/workPath.graphqls

```diff
@@ -0,0 +1,13 @@
+type WorkPath {
```

Comment:
Please add doc-comments to all fields in these 2 types

Reply:
all graphql representations are now commented